lib/python/test/KHI_growthRate/TestKHIManager.py

```diff
@@ -0,0 +1,440 @@
+
+        return:
+        -------
+        fieldData: list or array
+                   list or array of field Data
+        """
+
+        # Single key returns list
+        if(isinstance(key, str)):
+            return self.__khiData.getFieldSumByKey(key)
+
+        # List of keys returns array
+        else:
+            return [self.__khiData.getFieldSumByKey(name) for name in key]
+
+    def getGrowthRateByName(self, key):
+        """
+        returns growth rate for the passed fields
+
+        input:
+        -------
+        key:  str or list
+              sets the fields
+              (total, B_x, B_y, B_z, E_x, E_y, E_z)
+
+        return:
+        -------
+        fieldData: list or array
+                   list or array of growth rate
+        """
+
+        # Single key returns list
+        if(isinstance(key, str)):
+            return self.__khiData.getGrowthRateByKey(key)
+
+        # List of keys returns array
+        else:
+            return [self.__khiData.getGrowthRateByKey(name) for name in key]
+
+    def getAcceptanceLevel(self) -> float:
+        """
+        returns the acceptance level on which the assessment of the test
+        is based
+
+        return:
+        -------
+        acceptanceLevel:  float
+        """
+
+        return self.__acceptanceLevel
+
+    def getAcceptanceRange(self):
+        """
+        returns:
+        -------
+        acceptanceRange: tuple(start:float, end:float)
+        """
+
+        maxValue = self.getMaxGrowthRate()
+
+        return maxValue - (maxValue * self.__acceptanceLevel), maxValue + (
+                           maxValue * self.__acceptanceLevel)
+
+    def getTestResult(self) -> bool:
+        """
+        returns the result of the test, and thus whether the simulation
+        yields growth rates within the acceptance level
+
+        return:
+        -------
+        result: bool
+        """
+        if (abs(self.getMaxDifferenceInPercentage()) <=
+                self.__acceptanceLevel * 100):
+            return True
+        else:
+            return False
+
+    def getMaxDifferenceInPercentage(self, field: str = None) -> float:
+        """
+        indicates the difference (percentage) in the growth rate between the
+        transferred field from the simulation and the theoretical value
+
+        input:
+        -------
+        field:      str, optional, None
+                    ("total", "B_x", "B_y", "B_z", "E_x", "E_y", "E_z")
+                    if None the calculated field from the testsuite is used
+                    ("B_z" only for ESKHI, else "B_x")
+
+        return:
+        -------
+        differenz: float
+        """
+
+        if field is None:
+            field = self.__khiData.field
+
+        if self.__khiData.regime == "ESKHI":
+            maxValue = theoGR.getTheoryESKHI_max(
+                                            self.__khiData.gamma)
+        else:
+            maxValue = theoGR.getTheoryMIGR_max(
```

Comment:
I removed the function (it's still included in the source code in case I forgot to use the generalized one somewhere)

Reply:
If it is not supposed to be used anymore, please remove it completely. We do not want to have dead code lying around. It will just be a source of confusion in the future.

Reply:
Done